<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphStyleDialog</class>
 <widget class="QDialog" name="GraphStyleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graph Style</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/icons/chart-edit</normaloff>:/icons/chart-edit</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Data Type:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lblColumnType">
       <property name="text">
        <string>Temperature</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="leName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Line Style:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cbLineStyle">
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Line</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Step Left</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Step Right</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Step Center</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Impulse</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Point Style:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="cbPointStyle">
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dot</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cross</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/scatter_style/ssCross</normaloff>:/icons/scatter_style/ssCross</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Plus</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/scatter_style/ssPlus</normaloff>:/icons/scatter_style/ssPlus</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Circle</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/scatter_style/ssCircle</normaloff>:/icons/scatter_style/ssCircle</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Disc</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/scatter_style/ssDisc</normaloff>:/icons/scatter_style/ssDisc</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Square</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/scatter_style/ssSquare</normaloff>:/icons/scatter_style/ssSquare</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Diamond</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/scatter_style/ssDiamond</normaloff>:/icons/scatter_style/ssDiamond</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Star</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/scatter_style/ssStar</normaloff>:/icons/scatter_style/ssStar</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Triangle</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/scatter_style/ssTriangle</normaloff>:/icons/scatter_style/ssTriangle</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Triangle (Inverted)</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/scatter_style/ssTriangleInverted</normaloff>:/icons/scatter_style/ssTriangleInverted</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cross Square</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/scatter_style/ssCrossSquare</normaloff>:/icons/scatter_style/ssCrossSquare</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Plus Square</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/scatter_style/ssPlusSquare</normaloff>:/icons/scatter_style/ssPlusSquare</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cross Circle</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/scatter_style/ssCrossCircle</normaloff>:/icons/scatter_style/ssCrossCircle</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Plus Circle</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/scatter_style/ssPlusCircle</normaloff>:/icons/scatter_style/ssPlusCircle</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Line Colour:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QtColorButton" name="clrLineColour">
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>64</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtColorButton</class>
   <extends>QToolButton</extends>
   <header>qtcolorbutton/qtcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GraphStyleDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
